engine/api/workflow_queue.go

```diff
@@ -701,6 +701,15 @@ func (api *API) countWorkflowJobQueueHandler() service.Handler {
 func (api *API) getWorkflowJobQueueHandler() service.Handler {
 	return func(ctx context.Context, w http.ResponseWriter, r *http.Request) error {
 		since, until, limit := getSinceUntilLimitHeader(ctx, w, r)
+
+		status, err := QueryStrings(r, "status")
+		if err != nil {
+			return sdk.WrapError(err, "getWorkflowJobQueueHandler> Cannot parse request")
```

Comment:
in this case the error is not a sdk.Error and will be returned as a 500
you should do `sdk.NewError(sdk.ErrWrongRequest, err)`